package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "start": "npm run build && NODE_ENV=local node build/App.js",
     "build": "gulp build",
+    "watch": "npm run build:clean && gulp start:watch",
```

Comment:
Wouldn't `build` instead of `build:clean` be enough?

Reply:
Just for good measure I suppose